scripts/colmap2nerf.py

```diff
@@ -128,8 +128,11 @@ def qvec2rotmat(qvec):
 
 def rotmat(a, b):
 	a, b = a / np.linalg.norm(a), b / np.linalg.norm(b)
-	v = np.cross(a, b)
 	c = np.dot(a, b)
+	# handle exception for the opposite direction input
+	if(c < -1 + 1e-10):
```

Comment:
Before I merge, could you remove the parentheses to conform to the code style throughout the file? Thanks!

Reply:
Hi @Tom94,
Further inspection, I find that -eye cannot be a rotation matrix.
The rotation matrix(in fact, the so(3)) should always have its determinant as 1 but the -eye has -1.
I gonna try to find another solution.

Thank you.

Reply:
Hi @Tom94,
I solve that problem with add some uniform noise at reflection case.
you can check the mse is less or equal than normal rotation case and also the determinant is 1 at my [colab](https://colab.research.google.com/drive/1RlMqa-Mui4CWglwKJeMJnAQFdSKfbgfZ?usp=sharing).
After your review, I'll commit my code.

Thanks.

Reply:
Looks good to me, thank you!